### PR TITLE
Add a show title block

### PR DESCRIPTION
**Pull request by balderdash**
image

### CODE REVIEW

File: apps/src/block_utils.js [hunk 1]
```diff
@@ -426,6 +426,20 @@ const interpolateInputs = function (blockly, block, inputs) {
 };
 exports.interpolateInputs = interpolateInputs;
 
+/**
+ * Add pre-labeled inputs
```
Comment (Hamms) [inline apps/src/block_utils.js:430]: it's not that hard to infer, but I'd love some jsdoc documentation on the expected structure of the `args` param in particular

File: apps/src/block_utils.js [hunk 2]
```diff
@@ -488,6 +502,7 @@ exports.createJsWrapperBlockCreator = function (
     methodCall,
     eventBlock,
     eventLoopBlock,
+    inline,
```
Comment (Hamms) [inline apps/src/block_utils.js:513]: can this be documented, too?

File: apps/src/block_utils.js [hunk 3]
```diff
@@ -510,9 +525,13 @@ exports.createJsWrapperBlockCreator = function (
           });
         }
 
-        interpolateInputs(blockly, this, determineInputs(blockText, inputs));
+        if (inline === false) {
+          addInputs(blockly, this, args);
+        } else {
+          interpolateInputs(blockly, this, determineInputs(blockText, inputs));
+          this.setInputsInline(true);
+        }
```
Comment (Hamms) [inline apps/src/block_utils.js:541]: this changes the default behavior; will that break anything?

Comment (balderdash) [reply apps/src/block_utils.js:541]: For existing blocks that don't set the `inline` option, this is supposed to be exactly the same as before. How is it changing?

Comment (Hamms) [reply apps/src/block_utils.js:541]: Oh, I misread this!

I don't super love the two different expected values for the `inline` flag be `false` and `undefined`; falsiness tends to be fragile. But it does make sense for this application, so I'll say LGTM

**Pull request merged**